src/components/Login.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+export default function Login() {
+  interface User {
+    username: string;
+    password: string;
+  }
+  const [existingUser, setExistingUser] = useState<User>({
+    username: "",
+    password: "",
+  });
+  // if user does not exist in database
+  const [errorMsg, setErrorMsg] = useState("");
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
--- no action needed ---
i would prefer defining defining custom `handleChange` function for individual inputs, instead of creating a dependency with the html tag. 

i.e `<input name="username" onChange={ handleChangeUser } // handleChangeUser consumes the target value as username`
`<input name="password" onChange={ handleChangePassword } // handleChangePassword consumes the target value as password`